README.md

```diff
@@ -5,10 +5,10 @@
 [한국어](https://github.com/microsoft/vcpkg/blob/master/README_ko_KR.md)
 [Français](https://github.com/microsoft/vcpkg/blob/master/README_fr.md)
 
-Vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
-This tool and ecosystem are constantly evolving, and we always appreciate contributions!
+Vcpkg helps you manage C and C++ libraries on Windows, Linux, and macOS.
+This tool and ecosystem are constantly evolving, and we always appreciate your contributions!

```

Comment:
I don't think this part is correct, we appreciate contributions, not just "your contributions".